test/integration/crl_test.go

```diff
@@ -97,3 +101,84 @@ func TestCRLPipeline(t *testing.T) {
 	test.AssertEquals(t, string(reason), "5")
 	resp.Body.Close()
 }
+
+func TestTemporalAndExplicitShardingCoexist(t *testing.T) {
+	db, err := sql.Open("mysql", vars.DBConnSAIntegrationFullPerms)
+	if err != nil {
+		t.Fatalf("sql.Open: %s", err)
+	}
+	// Insert an old, revoked certificate in the certificateStatus table. Importantly this
+	// serial has the 7f prefix, which is in test/config-next/crl-updater.json in the
+	// `temporallyShardedPrefixes` list.
+	oldSerial := "7faa39be44fc95f3d19befe3cb715848e601"
```

Comment:
I think the comment should make it clear that this is a random serial, and doesn't correspond to some "real" cert that we know the integration tests have issued elsewhere.